src/app/app-routing.module.ts

```diff
@@ -59,6 +62,25 @@ const routes: Routes = [
     },
     {
         path: 'installation',
+        redirectTo: 'installation/presentation',
+    },
+    {
+        path: 'installation/presentation',
+        component: PresentationPageComponent,
+        canActivate: [VaultUninstalledGuardService],
+    },
+    {
+        path: 'installation/security',
+        component: SecurityPageComponent,
+        canActivate: [VaultUninstalledGuardService],
+    },
+    {
+        path: 'installation/hint',
+        component: HintPageComponent,
+        canActivate: [VaultUninstalledGuardService],
+    },
+    {
+        path: 'installation/installation',
```

Comment:
nit: maybe `installation/extension` would fit better the purpose of this page?

Reply:
Previously this step was named configureExtension, but now it also handles native app installation. So even if I am not very satisfied with installation/installation, I think it is still better. But I am very open...